packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -87,12 +87,12 @@ export class RelationshipsController extends TransportController {
     @log()
     private async updateExistingRelationshipInDb(id: string, response?: BackboneGetRelationshipsResponse) {
         const relationshipDoc = await this.relationships.read(id);
-        if (!relationshipDoc) throw CoreErrors.general.recordNotFound(Relationship, id);
+        if (!response || !relationshipDoc) throw CoreErrors.general.recordNotFound(Relationship, id);
```

Comment:
this is no typecheck, if you do it like this you invalidate the usage of this method without a response